### PR TITLE
Various adjustments to the code style (no functional changes)

### PR DESCRIPTION
**Pull request by lpugin**
* Use c++11 range loop when possible
* Use explicit types instead of auto for facilitating code understanding
* Revert changes yielding warnings with `size_t` change to `int`

### CODE REVIEW

File: src/vrv.cpp [hunk 1]
```diff
@@ -180,10 +180,8 @@ LogLevel StrToLogLevel(const std::string &level)
 
 bool LogBufferContains(const std::string &s)
 {
-    std::vector<std::string>::iterator iter = logBuffer.begin();
-    while (iter != logBuffer.end()) {
-        if ((*iter) == s) return true;
-        ++iter;
+    for (const std::string &logStr : logBuffer) {
+        if (logStr == s) return true;
```
Comment (DavidBauer1984) [inline src/vrv.cpp:184]: Looks much cleaner. Even nicer would be to use `std::any_of` here. But maybe that's something for another refactoring chapter.

**Review by DavidBauer1984 — APPROVED** [review]
Looks good as far as I can see. I've added some small comments with optional improvements.

File: src/vrv.cpp (hunk 1 above)
Comment (lpugin) [reply src/vrv.cpp:184]: Yes, I'll leave it like this.

File: src/chord.cpp [hunk 2]
```diff
@@ -515,7 +515,7 @@ int Chord::AdjustOverlappingLayers(const Doc *doc, const std::vector<LayerElemen
     int actualElementsInUnison = 0;
 
     // process each note of the chord separately, storing locations in the set
-    for (auto iter : notes) {
+    for (Object *iter : notes) {
```
[the remark this answers is not in the document]
Comment (lpugin) [reply src/chord.cpp]: Good point

**Pull request merged**